docs/Release-Guide.md

```diff
@@ -127,6 +127,27 @@ Bumping the limits may significantly improve build speed.
 * Publish message in Telegram at <https://t.me/VictoriaMetrics_en> and <https://t.me/VictoriaMetrics_ru1>
 * Publish message in Google Groups at <https://groups.google.com/forum/#!forum/victorametrics-users>
 
+## Operator
+
+The operator repository [https://github.com/VictoriaMetrics/operator/](https://github.com/VictoriaMetrics/operator/)
+
+### Bump the version of images
+
+- Bump `Version` field in [file `internal/config/config.go`](https://github.com/VictoriaMetrics/operator/blob/master/internal/config/config.go#L47) with new release version for:
+  - `vmalert` in `BaseOperatorConf.VMAlertDefault.Version`,
```

Comment:
shall we have a one place to change the version or it is possible to have different versions of components?

Reply:
I think not becasue:
- This is the default version, which can be overridden by the user for each component separately
- For some components a different version is specified (e.g. for vmbackup - enterprise)
- There may be a situation where different versions are really needed (e.g. in case of a critical bug or breaking change).